tests/maap_test_helper.py

```diff
@@ -12,6 +12,15 @@ def populate_maap_api_host(stage):
         env_prefix = f"{stage}."
     return f"api.{env_prefix}maap-project.org"
 
+def populate_maap_host(stage, prefix):
+    stage = stage if stage != "main" else "dit"
+    if stage == "production":
+        # env_prefix = "ops." after services are migrated, this should be ops
```

Comment:
I don't think this comment is true anymore? The plan for our "production" services now is to have them all hosted at a subdomain without a stage identifier (e.g. https://mmt.maap-project.org/). However, we do have the "ops" subdomains pointing at them via an alias record. See https://mmt.ops.maap-project.org/.

Reply:
Fixed